Source/WebGPU/WebGPU/BindGroupLayout.mm

```diff
@@ -136,7 +136,7 @@ static void addDescriptor(NSMutableArray<MTLArgumentDescriptor *> *arguments, MT
 static bool containsStage(WGPUShaderStageFlags stageBitfield, auto stage)
 {
     static_assert(1 == WGPUShaderStage_Vertex && 2 == WGPUShaderStage_Fragment && 4 == WGPUShaderStage_Compute, "Expect WGPUShaderStage to be a bitfield");
-    return stageBitfield & (1 << static_cast<uint32_t>(stage));
+    return (stageBitfield >> static_cast<uint32_t>(stage)) & 1;
```

Comment:
I'm not sure there is a benefit to this change, its less readable and when stage is a constant, before this change it would become:
`stageBitfield & (constant)`
now we have:
`(stageBitfield >> constant) & 1`

which doesn't seem like an improvement but really it probably makes no difference

Reply:
Stage is a variable. containsStage is not constexpr.